### DISCUSSION TITLE
[Help]: Health check endpoint?

### DISCUSSION THREAD

**Discussion by kevindurb** [mbentley/docker-omada-controller#352] [Help] [answered]
### Controller Version

5.12

### Describe Your Issue or Question

Hey i was just wondering if there is some kind of health check i could setup with the container? I have this container deployed on a k8s cluster and it would be sweet if i could get some kind of reporting if the container is not happy. I woke up this morning to getting 500s trying to access the web interface but the container was still "running" lol also i forgot to look at logs before i rerolled the deployment 🤦🏻‍♂️ 

### Expected Behavior

Health check endpoint? health check command?

### Steps to Reproduce

N/A

### How You're Launching the Container

```plain
k8s deployment
```

### Container Logs

```plain
N/A
```

**Comment by mbentley** [comment]
The [Dockerfile has a `HEALTHCHECK` defined](https://github.com/mbentley/docker-omada-controller/blob/master/Dockerfile.v5.x#L24) which calls a shell script but it's just doing a really simple wget command [as you can see here](https://github.com/mbentley/docker-omada-controller/blob/master/healthcheck.sh).  I have not checked to see if they have added any sort of actual ping or healthcheck endpoint to the app itself but loading to login page tends to work from what I can tell.

**Comment by mbentley** [comment, marked as answer]
If you want an endpoint that might be an even better health check that will verify that the MongoDB is running, check out the `/api/v2/anon/info` endpoint (i.e. - `https://localhost:8043/api/v2/anon/info`)
